src/test/java/core/basesyntax/files/ReaderFileImplTest.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.files;
+
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReaderFileImplTest {
+    private static final String SOURCE_FILE_GOOD = "src/main/resources/12-09-2021.csv";
+    private static final String SOURCE_FILE_BAD = "src/main/resources/sderfdsde.csv";
+    private Reader readerFileExist = new ReaderFileImpl(SOURCE_FILE_GOOD);
+    private Reader readerFileDoNotExist = new ReaderFileImpl(SOURCE_FILE_BAD);
```

Comment:
You can initialize it in the methods before Methods/Class. 
Note: It's a comment.